lisdf/parsing/parse_sdf.py

```diff
@@ -1,23 +1,52 @@
 import os
 
-from lisdf.parsing.sdf import SDF
+from lisdf.parsing.sdf import SDF, Collision, Link, Mesh, Visual
 
 
-def load_sdf(model_filename: str):
-    sdf_examples_dir = os.path.join(os.path.dirname(__file__), "../../models")
-    sdf_path = os.path.join(sdf_examples_dir, model_filename)
+def inject_absolute_path(sdf: SDF, model_path: str):
+
+    """
+    This function replace relative paths to object and material
+    files with absolute paths so the sdf object is self-contained.
+    """
+    for wi, world in enumerate(sdf.aggregate_order):
+        for model in world.models:
+            for component in model.aggregate_order:
+                if isinstance(component, Link):
+                    for link_component in component.aggregate_order:
+                        if isinstance(link_component, Collision):
+                            if (
+                                "geometry" in link_component.__dict__.keys()
```

Comment:
A `Collision` object is guaranteed to have a `self.geometry` but that might be None. What's the point of the check here?

Same story for `Visual`